sdk/highlight-node/src/client.ts

```diff
@@ -58,9 +105,8 @@ export class Highlight {
 	_projectID: string
 	_debug: boolean
 	otel: NodeSDK
-	private tracer: Tracer
+	tracer: Tracer
```

Comment:
rather than exposing the underlying otel tracer and having folks call `H.tracer.startActiveSpan`, we should have a helper `H.startSpan` like we do in other languages that calls `H.tracer.startActiveSpan` with the right args